pages/mesosphere/dcos/2.1/deploying-jobs/examples/index.md

```diff
@@ -23,7 +23,7 @@ These examples provide common usage scenarios for jobs.
 This JSON file creates a simple job with no schedule.
 
 1. Create a JSON file with the following contents.
-    ```json
+    ```json {data-line=2-5}
```

Comment:
@russurquhart1 @cmastr this is how you'd highlight lines. the rest of this PR is enablement.